apps/condo/pages/auth/change-password.tsx

```diff
@@ -62,6 +62,7 @@ const ChangePasswordPage: AuthPage = () => {
                 initialValues={initialValues}
                 colon={false}
                 style={{ marginTop: '40px' }}
+                requiredMark={false}                
```

Comment:
is it important?

Reply:
This flag only removes a red star from labels in required fields, the field still remains required. There are 3 options and they are strange:
1. By default = red star
2. false = hidden
3. optional = add word "optional" to all not required fields labels